isis/src/qisis/objs/CubeViewport/CubeViewport.cpp

```diff
@@ -200,7 +200,7 @@ namespace Isis {
 
     p_pixmapPaintRects = new QList<QRect *>();
     p_progressTimer = new QTimer();
-    p_progressTimer->setInterval(250);
```

Comment:
How does the interval relate to the double load issue?  I'm not super familiar with this

Reply:
The interval is basically how often to update the loading bar when loading images into qview. When set to 250, qview was waiting 0.25 seconds to update the loading bar. In most cases, this is very slow resulting in a less responsive feel to qview. Seems like making another PR would be a better way to go